proposals/new-project-donation-process.md

```diff
@@ -0,0 +1,69 @@
+# KEP-748: Project Donation to Kubeflow
+
+## Summary
+This KEP outlines how community members can donate their project to the Kubeflow community.
+
+## Motivation
```

Comment:
I think, we should also mention about the goal of Kubeflow Ecosystem to cover the entire AI/ML lifecycle, and how new projects can help to address missing stages.